examples/linear_model/plot_sgd_penalties.py

```diff
@@ -3,70 +3,48 @@
 SGD: Penalties
 ==============
 
-Plot the contours of the three penalties.
+Contours of where the penalty is equal to 1
+for the three penalties L1, L2 and elastic-net.
 
 All of the above are supported by
 :class:`sklearn.linear_model.stochastic_gradient`.
 
 """
-from __future__ import division
 print(__doc__)
 
 import numpy as np
 import matplotlib.pyplot as plt
 
+l1_color = 'C0'
+l2_color = 'C1'
+elastic_net_color = 'C2'
+
+line = np.linspace(-1.5, 1.5, 1001)
+xx, yy = np.meshgrid(line, line)
+
+l2 = xx ** 2 + yy ** 2
+l1 = np.abs(xx) + np.abs(yy)
+rho = 0.5
+elastic_net = rho * l1 + (1-rho)/2 * l2
+
+plt.figure(figsize=(10, 10), dpi=100)
+ax = plt.gca()
+
+elastic_net_contour = plt.contour(xx, yy, elastic_net, levels="1",
```

Comment:
I would replace `"1"` by `[1]`

Reply:
I put in the factor of  `\ 2` as according to here http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.ElasticNet.html
the regularization factor is
alpha * l1_ratio * ||w||_1 + 0.5 * alpha * (1 - l1_ratio) * ||w||^2_2

setting alpha to 1 and l1_ratio to rho gives what I used
but perhaps a different factor is used in SGD?

Reply:
Yes but L1 is `||w||_1` and L2 is `0.5 * ||w||^2_2`.

Reply:
In the figure, elastic-net line should be somewhere in between L1 and L2